TODO.md

```
@@ -20,4 +19,7 @@
 - propper dev enviroment with dev DB
 - possibly using admin / debug commands only in dev version of bot
 - schedule view handle pagination for 40+ components
-- reduce /profile commands, add interactive stuff
\ No newline at end of file
+- reduce /profile commands, add interactive stuff
+- improve ux with the roster update command (maybe a clickable link to prefill the command to add more chamions)
+    - alternative way to upload roster through web UI. 
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)